src/frontend/src/views/LandingPageV2/index.tsx

```diff
@@ -12,7 +12,7 @@ import UseCases from "./components/UseCases";
 import { Container, LoadingText } from "./style";
 
 export default function Homepage(): JSX.Element {
-  const { data: userInfo, isLoading } = useUserInfo();
+  const { data: userData, isLoading } = useUserInfo();
```

Comment:
I see sometimes you renamed to `user`, sometimes to `userData` and other times to `userInfo`. What's the pattern you used?

Reply:
There is no pattern 😅 -- I prefer the plain `user` but not sure if you have a different opinion? I'll update them all to be the same.

Reply:
I don't have a preference